sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -193,13 +193,47 @@ def value(self, new_value):
             pass
         self._value = new_value
 
+    @property
+    def filters(self):
+        # type: () -> List[Any]
+        if self._value is None:
+            self._value = {
+                "conditions": {
+                    "client_filters": [
+                        self._filters
+                    ]
+                }
+            }
+        elif isinstance(self._value, dict):
+            try:
+                self._value['conditions']['client_filters'] = self._filters
+            except KeyError:
+                self._value['conditions'] = {'client_filters': [self._filters]}
+        return self._filters
+
+    @filters.setter
+    def filters(self, new_filters):
+        if self._value is None:
+            self._value = {
+                "conditions": {
+                    "client_filters": [
+                        self._filters
+                    ]
+                }
+            }
+        elif isinstance(self._value, dict):
```

Comment:
If not isinstance(self._value, dict), your self._value will conflict with self._filters. Then what is the expected behavior for _to_generated()?

Reply:
My understanding is 
if user sets self._value to an non-dict, the object is not a FeatureFlagConfigurationSetting any longer. Hence if they want to get/set FeatureFlagConfigurationSetting properties, we raise AttributeError.